app/src/main/res/layout/fragment_main_current.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.layout.fragment_main_current` appears to be unused
 Unused resources make applications larger and slow down builds.